src/Toolkit/Properties.php

```diff
@@ -31,121 +28,25 @@ public function clone(array $props = [])
     }
 
     /**
-     * Creates a clone and fetches all
-     * lazy-loaded getters to get a full copy
-     *
+     * @deprecated Use `::clone()` instead
+     * @todo Remove in 3.7.0
```

Comment:
We use `hardcopy()` in the `commit()` methods of our models at the moment.
The question is: What did this method actually do? The return value of `propertiesToArray()` isn't used, so does it entirely rely on possible side-effects of getters? If so, do we use this behavior anywhere at the moment?

Reply:
@bastianallgeier will need to say where this behaviour was actually used. I added another approach that might work in the same way though.